src/test/groovy/graphql/validation/rules/DeferDirectivesTest.groovy

```diff
@@ -0,0 +1,148 @@
+package graphql.validation.rules
+
+import graphql.language.Document
+import graphql.parser.Parser
+import graphql.validation.LanguageTraversal
+import graphql.validation.RulesVisitor
+import graphql.validation.SpecValidationSchema
+import graphql.validation.TraversalContext
+import graphql.validation.ValidationContext
+import graphql.validation.ValidationError
+import graphql.validation.ValidationErrorCollector
+import graphql.validation.ValidationErrorType
+import graphql.validation.Validator
+import spock.lang.Specification
+
+class DeferDirectivesTest extends Specification {
+
+    ValidationContext validationContext = Mock(ValidationContext)
+    ValidationErrorCollector errorCollector = new ValidationErrorCollector()
+
+    DeferDirectiveLabel deferDirectiveLabel = new DeferDirectiveLabel(validationContext, errorCollector)
+
+    def setup() {
+        def traversalContext = Mock(TraversalContext)
+        validationContext.getSchema() >> SpecValidationSchema.specValidationSchema
+        validationContext.getTraversalContext() >> traversalContext
+    }
+
+    def "Allow unique label directive"() {
+        given:
+        def query = """
+            query defer_query {
+              ... @defer(label: "name") {
```

Comment:
I'm missing tests that pass `label` as a variable.
Actually, validation should fail if `label` is passed as a variable.
The spec says:
```
If a label is passed, it must not be a variable
```